src/app/zap-templates/zcl/data-model/chip/mode-select-cluster.xml

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 <configurator>
   <domain name="CHIP"/>
 
-  <struct name="SemanticTag">
+  <struct name="SemanticTagStruct">
     <cluster code="0x0050"/>
     <item name="MfgCode" type="ENUM16"/>
```

Comment:
My copy of the spec has ID0 for value and ID1 for MfgCode, which seems to imply this struct order needs to be reversed?

Or do we need to fix the spec if this was 1.0 released?

Reply:
Oh, nice catch...  Yes, this is broken and at this point that means it needs to be fixed in the spec.  Filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6165